css/modal.css

```diff
@@ -0,0 +1,205 @@
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1000;
+  padding-top: 100px;
+  padding-bottom: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
```

Comment:
- [ ] Kindly make sure you add the **blur** effect on the background color of your **Modal Popup** to match the Figma design.
#### Your Background view
![Screenshot from 2022-09-07 15-22-24](https://user-images.githubusercontent.com/83042742/188889203-5b383936-d4d4-43a2-8ea4-f720fefddcb1.png)

#### Figma view
![Screenshot from 2022-09-07 15-23-54](https://user-images.githubusercontent.com/83042742/188889369-135243cf-828e-4c23-a15e-15d97d461d34.png)